Assets/MRTK/Core/Utilities/Editor/SpatializerUtilities.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Editor;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
+{
+    /// <summary>
+    /// Collection of utilities to manage the configured audio spatializer.
+    /// </summary>
+    public static class SpatializerUtilities
+    {
+        /// <summary>
+        /// Returns the name of the currently selected spatializer plugin.
+        /// </summary>
+        public static string CurrentSpatializer => AudioSettings.GetSpatializerPluginName();
+
+        /// <summary>
+        /// Returns the names of installed spatializer plugins.
+        /// </summary>
+        public static string[] InstalledSpatializers => AudioSettings.GetSpatializerPluginNames();
+
+        /// <summary>
+        /// Checks to see if the audio spatializer is configured and/or whether or
+        /// not the spatializer collection has changed.
+        /// </summary>
+        /// <returns>
+        /// True if; the selected spatializer is installed and no changes have been made to the collection of installed spatializers.
+        /// False if; There is no selected spatializer, the selected spatializer is no longer installed or the collection of installed spatializers has been changed.
+        /// </returns>
+        public static bool CheckSettings()
+        {
+            string spatializerName = CurrentSpatializer;
+
+            // Check to see if an audio spatializer is configured.
+            if (string.IsNullOrWhiteSpace(spatializerName)) 
+            {
+                // A spatializer has not been configured.
+                return false; 
+            }
+
+            string[] installedSpatializers = AudioSettings.GetSpatializerPluginNames();
```

Comment:
Why not use the public property InstalledSpatializers?

Reply:
Updated